app/javascript/components/pages/PostIndex.js

```diff
@@ -7,22 +7,28 @@ import {
 	Button,
 } from 'reactstrap'
 import Footer from '../components/Footer'
+import galaxy from '../assets/galaxy.jpg'
 
 
 class PostIndex extends Component {
 	render() {
 		const { posts } = this.props
 		return (
 			<React.Fragment>
-				<h1 style={{ textAlign: 'center', fontSize: '5rem' }}>All the Posts</h1>
-				<hr
+
+                <div>
+                <img src={galaxy} className="bg"></img>
+                
+                </div>
+				<h1 className= 'title-formating'>All the Posts</h1>
+				{/* <hr
 					style={{
 						height: '.5rem',
 						border: 'none',
-						color: '#212529',
-						backgroundColor: '#212529',
+						color: '#fafeff',
```

Comment:
it seems strange to have color and background color be the same. text won't be visible.  I know this is commented code, but just for future reference